packages/server/modules/core/repositories/streams.js

```diff
@@ -0,0 +1,216 @@
+const _ = require('lodash')
+const { Streams, StreamAcl, StreamFavorites, knex } = require('@/modules/core/dbSchema')
+const { InvalidArgumentError } = require('@/modules/core/errors/base')
+
+/**
```

Comment:
Split all data access code to the repository layer for a better separation of concerns. Similarly moved all logic away from the resolvers to the service layer.

So layers are utilized like this: Resolver > Service > (DataLoader) > Repository